WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -1280,6 +1283,10 @@ class MySiteViewModel @Inject constructor(
         }
     }
 
+    private fun onBloggingPromptViewMoreClicked() {
+        _onBloggingPromptsViewMore.postValue(Event(Unit))
```

Comment:
Same question about `.value = ` instead of `postValue`.

Reply:
Same here, thanks!